src/app-gocardless/banks/ssk-dusseldorf-dussdeddxxx.js

```diff
@@ -0,0 +1,53 @@
+import Fallback from './integration-bank.js';
+
+import { formatPayeeName } from '../../util/payee-name.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['SSK_DUSSELDORF_DUSSDEDDXXX'],
+
+  accessValidForDays: 90,
+
+  /**
+   * Following the GoCardless documentation[0] we should prefer `bookingDate`
+   * here, though some of their bank integrations uses the date field
+   * differently from what's described in their documentation and so it's
+   * sometimes necessary to use `valueDate` instead.
+   *
+   *   [0]: https://nordigen.zendesk.com/hc/en-gb/articles/7899367372829-valueDate-and-bookingDate-for-transactions
+   */
```

Comment:
I think this was copied from another bank handler, it can probably be omitted here

```suggestion
```